pypy/module/faulthandler/cintf.py

```diff
@@ -26,7 +26,7 @@ def direct_llexternal(*args, **kwargs):
 
 
 DUMP_CALLBACK = lltype.Ptr(lltype.FuncType(
-                     [rffi.INT, rffi.SIGNEDP, lltype.Signed], lltype.Void))
+                     [rffi.INT_real, rffi.VOIDP, rffi.INT_real], lltype.Void))
```

Comment:
What we really want here is a platform `intptr_t`, but it seems hard to get the lltype.Number and genc to actually output `intptr_t` instead of `Signed`.